sequencer.c

```diff
@@ -866,6 +877,18 @@ static char *get_author(const char *message)
 	return NULL;
 }
 
+static char *read_author_date_or_die(void)
+{
+	char *date;
+
+	if (read_author_script(rebase_path_author_script(),
+			       NULL, NULL, &date, 0))
+		die(_("failed to read author date"));
+
+	/* needs to be free()'d by the caller */
```

Comment:
I added this comment so that devs need not dig any deeper to know if this needs to be free()'d.

Reply:
Yeah I think it would be nicer to have this above the function, so readers can see immediately what's happening, without reading the code.

The convention in C is usually that things returned as `char *` require `free`ing, while things returned as `const char *` don't.  But I think the comment can still help people not familiar with the convention.